temoa/extensions/stochastics/stochastic_sequencer.py

```
@@ -0,0 +1,108 @@
+from temoa.core.config import TemoaConfig
+from temoa.extensions.stochastics.stochastic_config import StochasticConfig
+
+logger = logging.getLogger(__name__)
+
+
+class StochasticSequencer:
+    """
+    Orchestrates a stochastic run using mpi-sppy.
+    """
+
+    def __init__(self, config: TemoaConfig):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)